src/actions.sol

```diff
@@ -118,6 +118,16 @@ contract Actions {
         return loan;
     }
 
+    function issue(address shelf_, address registry, uint256 token) public onlyDelegateCall returns (uint256 loan) {
```

Comment:
I don't think we need to re-add this. It just matters that the functions which are used, have the same interface as before.